clean, clean, clean (#1901)
enkidevs/curriculum: javascript/core/functions-ii/functions-are-objects.md

Every function in JavaScript is also an object. 

This means we can assign a function to a variable which can then be used to execute the function. Function used as a value like this is called a *function expression*.

```js
let add = function enki(x, y){
  return x + y;
};

add(1, 2); // 3
```

In fact, we can think of functions as invocable (executable) objects.

This means that we can also pass function as arguments into other functions (the passed in functions are usually called "callbacks").

> A somewhat contrived example would be, if we were creating a calculator, we could pass different calculation expressions into the `calc` function and avoid repeating the console logging code:

Reviewer: feel like the phrase was a bit crowded. how does this sound?
```suggestion
Consider this example: if we were creating a calculator, we could pass different calculation expressions into the `calc` function and avoid repeating the console logging code:
```
